core/templates/dev/head/pages/about/about.html

```diff
@@ -41,7 +41,7 @@
           <div class="about oppia-about-tab-content oppia-about-visible-content">
             <div class="oppia-static-card-content oppia-static-card-content-narrow">
               <h2>About Oppia</h2>
-              <div class="pull-right">
+              <div class="oppia-about-text-align">
                 <em>Oppia "O-pee-yah" (Finnish) - "to learn"</em>
               </div>
               <br>
```

Comment:
The pull-right class floated to the right, which eliminated some of the visible spacing. We can now remove this `<br>` because the spacing will be correct without it. Aside from that, LGTM!